### PR TITLE
Task-39856: XSS issues in differents places (#199)

### PR DESCRIPTION
**Pull request by mkrout**

### CODE REVIEW

File: services/src/main/java/org/exoplatform/task/util/StorageUtil.java [hunk 1]
```diff
@@ -67,7 +72,13 @@ public static TaskDto taskToDto(Task taskEntity,ProjectStorage projectStorage) {
         TaskDto task = new TaskDto();
         task.setId(taskEntity.getId());
         task.setTitle(taskEntity.getTitle());
-        task.setDescription(taskEntity.getDescription());
+        if(taskEntity.getDescription()!=null) {
+            try {
+                task.setDescription(HTMLSanitizer.sanitize(taskEntity.getDescription()));
+            } catch (Exception e) {
+                LOG.warn("Task description cannot be sanitized");
```
Comment (plamarque) [inline services/src/main/java/org/exoplatform/task/util/StorageUtil.java]: is it a good idea to swallow the exception here ?

Comment (mkrout) [reply services/src/main/java/org/exoplatform/task/util/StorageUtil.java]: If the description can't be sanitized the field will be empty, I don't see anything else to do in this case, imo, that should not prevent the task to be displayed

Comment (plamarque) [reply services/src/main/java/org/exoplatform/task/util/StorageUtil.java]: I mean for diagnostic. we won't be able to troubleshoot what's wrong with the sanitizer. we'd know it fails, but no clue about why

Comment (mkrout) [reply services/src/main/java/org/exoplatform/task/util/StorageUtil.java]: You're right, A add the exception to the log

**Review by plamarque — COMMENTED** [review]
LGTM

**Review by lbengayed — APPROVED** [review]

**Pull request merged**